src/shared/api/handle-param-scope.ts

```diff
@@ -0,0 +1,34 @@
+import type { Reactive } from 'vue';
+import { computed, reactive } from 'vue';
+import { useErrorHandlers } from '@/shared/ui/composables/useErrorHandlers';
+import { useErrorRetry, useTask, wheneverRejected } from '@vue-kakuyaku/core';
+
+export function handleParamScope<T, K>(
```

Comment:
`handle` is not a conventional prefix for composables (which this function is). More over, this function doesn't set up a scope (doesn't rely on EffectScope API). It is a plain composable that sets up some reactive logic.

Consider other options:

- `useAsyncData`
- `setupAsyncData`